cmd/hub/cli/cli.go

```diff
@@ -54,6 +54,8 @@ func Init(rootCmd *cobra.Command) {
 	rootCmd.AddCommand(bucketCmd)
 	buck.Init(bucketCmd)
 
+	versionCmd.Flags().BoolP("update", "u", false, "Update hub cli to latest release")
```

Comment:
Feels to me like `update` should be its own command. Thoughts?

Reply:
i've got the least opinion here. i had originally intended to go that way, but then felt that the nav felt cleaner with just a version command

Reply:
K, cool. My two cent: We still have relatively few commands... and putting it on the top level will make it more noticeable as well